projects/storefrontlib/src/cms-components/product/product-variant-selector/product-variant-selector.component.ts

```diff
@@ -0,0 +1,32 @@
+import { ChangeDetectionStrategy, Component, OnInit } from '@angular/core';
+import { Product, BaseOption, VariantType } from '@spartacus/core';
+import { Observable } from 'rxjs';
+import { CurrentProductService } from '../current-product.service';
+import { tap, filter, distinctUntilChanged } from 'rxjs/operators';
+
+@Component({
+  selector: 'cx-product-variant-selector',
+  templateUrl: './product-variant-selector.component.html',
+  changeDetection: ChangeDetectionStrategy.Default,
```

Comment:
We don't need `ChangeDetectionStrategy.Default` probably, btw, is it possible to make it OnPush?